### ISSUE TITLE
test problems on linux

### ISSUE THREAD

**Issue by faddat** [Stride-Labs/stride#333]
## Summary of Bug

Tests are hard to replicate on linux

## Version

all

## Steps to Reproduce

make build-docker
make test-localnet-docker

## Additional context

https://docs.docker.com/build/buildkit/#getting-started

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by nghuyenthevinh2000** [comment]
Right, it is a bit hard for me to follow what linux version is breaking. Probably should put a list here:
I don't know if there is more

- [ ] Arch linux
- [ ] Ubuntu 

cc @asalzmann
